pkg/kv/params/database.go

```diff
@@ -53,4 +53,9 @@ type DynamoDB struct {
 	AwsProfile         string
 	AwsAccessKeyID     string
 	AwsSecretAccessKey string
+
+	// Interval to run health check for the DynamoDB instance
+	// Won't run when is equal or less than 0
+	// Default: 0
```

Comment:
1. code doc format
2. think it is maybe too much documentation - keep it here or on the configuration (here is fine)
3. remove the default - it is based on the configuration.